models/DataObject/Localizedfield.php

```diff
@@ -402,6 +427,10 @@ public function setLocalizedValue($name, $value, $language = null)
             );
         }
 
+        //TODO implement isEqual for as many datatypes as possible (at least the rather simple ones)
```

Comment:
todo still open?

Reply:
not really, it is up to the data type if it wants to contribute to the dirty detection. if it doesn't it is dirty by default. I implemented it for most of the simple built-in datatypes.